playbooks/azure/openshift-cluster/example_params.yml

```diff
@@ -0,0 +1,35 @@
+---
+# need these to connect with a service principal
+# this didn't seem to work, had to resource to ~/.azure/credentials file
```

Comment:
Unclear what you did here. How can I get this file down? Is it the output of `az login`?

Reply:
I created a credentials file with a default profile:
```
$ cat ~/.azure/credentials 
[default]
client_id=<your client id>
secret=<your secret>
subscription_id=<your sub id>
tenant=<your tenant id>
```

and then set the env var AZURE_PROFILE=default

Subscription and tenant id are available in `az account show` as `id` and `tenantId`.  Client id and secret are from the service principal.